substra/sdk/client.py

```diff
@@ -110,6 +110,20 @@ def set_user(self):
                 })
                 self.client.set_config(self._current_profile, self._profile_name)
 
+    def update_permissions_field(self, data, permissions_field='permissions'):
+        if permissions_field:
```

Comment:
The purpose of this function is to provide a default value for a permissions field. It shouldn't have to handle the case in which there is no permission field at all. It is the job of the parent function to decide whether to call this function or not.

Reply:
I'm agree!